recipes/zlib-ng/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "2.0.2":
+    sha256: "dd37886f22ca6890e403ea6c1d60f36eab1d08d2f232a35f5b02126621149d28"
+    url: "https://github.com/Dead2/zlib-ng/archive/2.0.2.tar.gz"
```

```suggestion
    url: "https://github.com/Dead2/zlib-ng/archive/2.0.2.tar.gz"

```

Conan hooks requires empty end-line